services/ui/ws/window_tree.cc

```diff
@@ -2465,6 +2466,14 @@ void WindowTree::PerformWindowMove(uint32_t change_id,
     return;
   }
 
+  if (window_server_->IsInExternalWindowMode()) {
+    // In external window mode, we just return early, which means an access to
+    // perform the window move is allowed.
```

Comment:
nit: the part "which means an access to perform the window mode" isnt too clear in my mind. maybe replace "access" with something else?